pkgs/top-level/python-packages.nix

```diff
@@ -4136,6 +4148,12 @@ in {
     inherit python;
   }));
 
+  fontPens = callPackage ../development/python-modules/fontPens { };
+
+  fontMath = callPackage ../development/python-modules/fontMath { };
+
+  fontParts = callPackage ../development/python-modules/fontParts { };
```

Comment:
Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)